src/components/CheckIn/CheckInItems.js

```diff
@@ -156,7 +168,8 @@ class CheckInItems extends Component {
       'comment': piece => piece.comment,
       'pieceStatus': piece => piece.receivingStatus,
     };
-    const isCheckInDisabled = !items.some(piece => piece.isChecked === true);
+    const pieces = items.filter(item => item.isChecked === true);
```

Comment:
I think `isChecked` is boolean, why do you need `=== true`?

Reply:
Fixed